triaging.rst

```diff
@@ -375,6 +375,18 @@ Various informational flags about the issue. Multiple values are possible.
 | easy          | Fixing the issue should not take longer than a day for     |
 |               | someone new to contributing to Python to solve.            |
 +---------------+------------------------------------------------------------+
+| easy (C)      | Fixing the issue should not take longer than a day for     |
+|               | someone new contributing to Python, focused on C.          |
++---------------+------------------------------------------------------------+
+| security_issue| The issue would fit as, or is related as a security issue. |
+|               | The main difference from "security" is that this is a      |
+|               | definite security problem that has to be dealt with.       |
++---------------+------------------------------------------------------------+
+| PEP 3121      | The issue fit as, or is related to the PEP 3121 module     |
+|               | Which is the Extension Module Initialization and           |
+|               | Finalization, For More information:                        |
+|               | https://www.python.org/dev/peps/pep-3121/                  |
```

Comment:
The wording and capitalization here are a bit clumsy. My suggestion:

"The issue is related to \`PEP 3121\`_: Extension Module Initialization and Finalization."

(The above assumes adding the link for PEP 3121 below it.)

Reply:
I have made the requested changes; please review again